spec/features/checkout_spec.rb

```diff
@@ -1,13 +1,22 @@
 require 'spec_helper'
 
 feature 'checkout' do
+  let(:user) { create(:user) }
+  let(:order) { OrderWalkthrough.up_to(:delivery) }
+  let(:sisow_request_url) { 'http://www.sisow.nl/Sisow/iDeal/RestHandler.ashx/TransactionRequest' }
```

Comment:
Line is too long. [99/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.